ReadMe.md

```diff
@@ -1,3 +1,116 @@
 # MAGE SDK for JavaScript (ES5)
```

Comment:
Having said that, for now, we only support browsers. So for now, we should label it Web.